subsys/net/lib/nrf70_fw_ext/rpu_fw_patches.ld

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/**
+ * @file
+ * @brief Custom Linker command/script file
+ *
+ * Custom Linker script for the Cortex-M platforms.
+ */
+
+#include <zephyr/linker/sections.h>
+#include <zephyr/devicetree.h>
+
+#include <zephyr/linker/linker-defs.h>
+#include <zephyr/linker/linker-tool.h>
+
+#if CONFIG_BOARD_NRF5340DK_NRF5340_CPUAPP || CONFIG_BOARD_NRF52840DK_NRF52840
+/*
+ * nRF53/52 series ship an external flash that can be used for XIP using QSPI/SPI.
+ *
+ * Note: In nRF7002 external flash using is accessible only using SPI but there is no
+ *       support for XIP, so, relocation cannot be used.
+ */
+#if CONFIG_BOARD_NRF5340DK_NRF5340_CPUAPP
+#define EXTFLASH_BASE_ADDR 0x10000000
+#define EXTFLASH_SIZE 0x800000
+#elif CONFIG_BOARD_NRF52840DK_NRF52840
+#define EXTFLASH_BASE_ADDR 0x12000000
+#define EXTFLASH_SIZE 0x800000
+#endif /* CONFIG_BOARD_NRF5340DK_NRF5340_CPUAPP */
+
+#if USE_PARTITION_MANAGER && PM_EXTERNAL_FLASH_ADDRESS
+#include <pm_config.h>
+#define EXTFLASH_ADDRESS (EXTFLASH_BASE_ADDR + PM_EXTERNAL_FLASH_ADDRESS)
+#undef EXTFLASH_SIZE
+#define EXTFLASH_SIZE (PM_EXTERNAL_FLASH_SIZE)
+#else
+#define EXTFLASH_ADDRESS (EXTFLASH_BASE_ADDR)
+#endif /* USE_PARTITION_MANAGER && PM_EXTERNAL_FLASH_ADDRESS */
```

Comment:
this should really have it's own PM section that is dedicated, right now it looks like it's reading all of the external qspi flash

Reply:
Sure, will take it up separately in another PR as this was a old file.